lib/atat-web-api-stack.ts

```diff
@@ -42,5 +56,25 @@ export class AtatWebApiStack extends cdk.Stack {
         })
       )
     );
+
+    // State Machine
+    const provisioningSfn = new ProvisioningWorkflow(this, { environmentName });
+    this.provisioningStateMachine = new StateMachine(this, "ProvisioningStateMachine", {
+      stateMachineProps: {
+        definition: provisioningSfn.workflow,
+      },
+      logGroup: provisioningSfn.logGroup,
+    }).stateMachine;
+
+    // Provisioning lambda that translate and invokes the state machine
+    const provisioningJob = new ApiSfnFunction(this, "ProvisioningJobRequest", {
+      method: HttpMethod.POST,
+      handlerPath: "api/provision/start-provision-job.ts",
+      stateMachine: this.provisioningStateMachine,
+    });
+
+    // APIGW Provisioning Job Resource
+    const provisioningJobsResource = apigw.restApi.root.addResource("provisioning-job");
+    provisioningJobsResource.addMethod(provisioningJob.method, new LambdaIntegration(provisioningJob.fn));
```

Comment:
To match case of route
```suggestion
    const provisioningJobResource = apigw.restApi.root.addResource("provisioning-job");
    provisioningJobResource.addMethod(provisioningJob.method, new LambdaIntegration(provisioningJob.fn));
```